### DISCUSSION TITLE
Wait for all http requests to finish

### DISCUSSION THREAD

**Discussion by roydekleijn** [cypress-io/cypress#23568] [General]
Hello,

I'm testing a lowcode application. After certain actions (enter text in input, click on certain elements) zero/one/multiple requests are performed to the server. I like to wait till those requests are done. I know I can use:

```
 cy.intercept('POST', '/xas/').as('ajaxRequest')

 cy.wait('@ajaxRequest')
```
 
 This works if you know the amount of requests upfront. But in this application we don't know that. Ideally, I implement something generic that can wait for zero/one/multiple /xas/ requests.
 
 How would that be possible? Any help is much appreciated!
Roy

**Comment by palamarchukser** [comment]
https://stackoverflow.com/questions/68359997/cy-intercept-multiple-matchers-found-is-it-possible-to-only-wait-for-the-last-n
